src/components/EventDetails.tsx

```diff
@@ -0,0 +1,73 @@
+import { ReactElement } from "react";
+import Frame from ".././assets/Frame.svg";
+import MeatballMenuSVG from ".././assets/MeatballMenu.svg";
+import { Event } from "./EventCard";
+
+export type EventDetailsProps = {
+  event: Event;
+};
+
+const EventDetails = ( {event}:EventDetailsProps ): ReactElement => {
+  return (
+    <div
+      className="flex h-screen w-screen flex-col p-10"
+      style={{ gap: "2rem" }}
+    >
+      <div className="flex flex-row items-center" style={{ gap: "1rem" }}>
+        <img src={Frame} alt="Back arrow" className="p-1 pt-0" />
+
+        <h1 className="section-heading m-0 text-md text-left w-full md:w-2/12">
+          <span>EVENT DETAILS</span>
+        </h1>
+        <img
+          src={MeatballMenuSVG}
+          alt="Menu svg"
+          aria-label="Open Event Card details"
+        />
+      </div>
+      <hr className="border-black home-mx -ml-0 -mr-0" />
+      <div className="flex flex-row ">
+        <div className="not-italic font-bold text-2xl leading-8 font-sans break-words ">
+          Sample Event
+        </div>
+      </div>
+      <div
+        className="flex flex-col w-full justify-center md:justify-start "
+        style={{ gap: "1rem" }}
+      >
+        <div className="flex flex-row md:flex-col">
+          <div
```

Comment:
if an element has no children elements, you can just self-close it like this: `<div />`

Reply:
Also I'm wondering if this might be better as an `img` rather than a `div` to make it more clear that it's an image. If it doesn't work with `img` it'd be fine to keep it `div`!